jujugui/static/gui/src/app/app.js

```diff
@@ -763,7 +763,9 @@ YUI.add('juju-gui', function(Y) {
           instanceName="inspector-panel"
           visible={true}
           metadata={metadata}>
-          <components.Inspector />
+          <components.Inspector>
+            <components.ServiceOverview />
```

Comment:
I don't think that this should go here because this will render the `ServiceOverview` every time this method is called. But we actually only want to render the `ServiceOverview` when state indicates that the inspector should be in the default state.